utils/sl/prereq_tools/base.py

```diff
@@ -1562,8 +1568,8 @@ def _rm_old_dir(self, path):
         if self.__dry_run:
             print('Would empty %s' % path)
         else:
-            os.system("rm -rf %s" % path)
-            os.mkdir(path)
+            subprocess.call(["/bin/rm", "-rf", path])
+            subprocess.call(["/bin/mkdir", path])
```

Comment:
These should probably use the python native functions (untested)
```suggestion
            shutil.rmtree(path)
            os.mkidr(path)
```

Reply:
Fixed.